core/src/Microsoft.Mcp.Core/Configuration/McpServerConfiguration.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Mcp.Core.Configuration;
+
+public sealed class McpServerConfiguration
+{
+    public const string DefaultName = "Microsoft.Mcp.Server";
```

Comment:
This base class shouldn't set a default server name.  Each implementation would probably needs its own configuration class where the default name is set.